site/content/en/docs/start/_index.md

```diff
@@ -20,6 +20,47 @@ All you need is Docker (or similarly compatible) container or a Virtual Machine
 
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
 
+<script language="javascript">
+  const architectures = [
+    "Linux/x86-64",
+    "Linux/ARM64",
+    "Linux/ppc64",
+    "Linux/S390x",
+    "Linux/ARMv7",
+    "macOS/x86-64",
+    "macOS/ARM64",
+    "Windows/x86-64"
+  ];
+
+  fetch('https://api.github.com/repos/kubernetes/minikube/releases')
```

Comment:
can we console out log how long this takes, so for reference ? I wanna know if it worth it to make our site slower by this call ..

Reply:
I added a timer to measure the performance. Anyway it's asynchronous, it shouldn't affect page loading 🙂 .